frontend/src/pages/MyTemplatesPage/MyTemplatePage.tsx

```diff
@@ -16,11 +16,15 @@ const MyTemplatePage = () => {
   const [selectedCategoryId, setSelectedCategoryId] = useState<number | undefined>(undefined);
   const [page, setPage] = useState<number>(1);
   const [pageSize] = useState<number>(20);
+  const [keyword, setKeyword] = useState<string>('');
```

Comment:
기존에 만들어두었던 `useInput` 훅을 사용해도 좋을 것 같아요! 
그러면 굳이 아래에서 `handleSearchChange` 도 만들어주지 않아도 됩니다.

```suggestion
  const [keyword, handleKeyword] = useInput('');
```